test/passes/strip-target-features_roundtrip_print-features_all-features.wast

```diff
@@ -0,0 +1,14 @@
+;; Test that features survive a round trip even if the target features
```

Comment:
"features survive" => "the IR representation of enabled features on the Module survive" perhaps? otherwise one might think it's the feature section or the instructions using a feature or something else, maybe.